### PR TITLE
Add scripts to evaluate models in the zoo on different datasets

### PR DESCRIPTION
**Pull request by fengyuentau**
- [x] Evaluation framework
- [x] Initial dataset for evaluation: ImageNet
- [x] Report mobilenet accuracy
- [x] Report ppresnet accuracy

Note: the accuracy of PP-ResNet quantized is very low, which seems to be bad quantization.

### CODE REVIEW

File: tools/eval/eval.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+datasets = dict(
+        imagenet=dict(
+            name="ImageNet",
+            topic="image_classification",
+            size=224),
+)
+
+def main(args):
+    # Instantiate model
```
Comment (zihaomu) [inline tools/eval/eval.py:64]: At the beginning of the script we can print the model name and test data home name.

**Review by zihaomu — APPROVED** [review]
Thank you! 👍

File: tools/eval/eval.py (hunk 1 above)
Comment (fengyuentau) [reply tools/eval/eval.py:64]: Such information is printed in `dataset.eval()` like the following:

image

**Comment by fengyuentau** [comment]
The issue of low accuracy of quantized PP-ResNet will be fixed in another pull request. Merge as is.

**Pull request merged**